src/main/java/com/conveyal/taui/models/Bundle.java

```diff
@@ -67,15 +57,27 @@ public Bundle clone () {
         public LocalDate serviceEnd;
         public long checksum;
 
+        /**
+         * Set service start and end from the dates of service values returned from GTFSFeed. This is calculated from
+         * the trip data in the feed. Feed summaries (feed_start_date/feed_end_date) are not always included in feeds.
```

Comment:
Please clarify the meaning of "feed summaries" in this context. Does it mean FeedSummary objects? Or does it mean the declared feed start/end date from feed_info.txt? Please state that clearly, e.g. "feed_info.txt is optional and many GTFS feeds do not include the fields feed_start_date or feed_end_date. Therefore we instead derive the start and end dates from the stop_times and trips." However I'm not sure whether we should be outright ignoring the declared range of valid dates for a feed. Feeds frequently contain data outside their real valid date range, especially when they're the result of merging more than one feed.

Reply:
The only usage of these fields is to explicitly show in the user interface that a date does or does not have service. There are lots more specific data we could bubble up, but the only intention is to catch when the user mistakenly put in a totally incorrect date. The exact bundle contents and feeds within them are still up for the user to know. I'll update the the comments with a longer description.